src/drivers/bitbucket_cloud.js

```diff
@@ -118,19 +119,107 @@ class BitbucketCloud {
   }
 
   async runnerToken() {
-    throw new Error('Bitbucket Cloud does not support runnerToken!');
+    return '';
+  }
+
+  async startRunner(opts) {
+    const { projectPath } = this;
+    const { name, labels } = opts;
+
+    try {
+      const { uuid: accountId } = await this.request({ endpoint: `/user` });
+      const { uuid: repoId } = await this.request({
+        endpoint: `/repositories/${projectPath}`
+      });
+      const {
+        uuid,
+        oauth_client: { id, secret }
+      } = await this.registerRunner({ name, labels });
+
+      const gpu = await gpuPresent();
+      const command = `docker container run -t -a stderr -a stdout --rm \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      -v /var/lib/docker/containers:/var/lib/docker/containers:ro \
+      -v /tmp:/tmp \
```

Comment:
```suggestion
    const { projectPath } = this;
    const { workdir, name, labels } = opts;

    try {
      const { uuid: accountId } = await this.request({ endpoint: `/user` });
      const { uuid: repoId } = await this.request({
        endpoint: `/repositories/${projectPath}`
      });
      const {
        uuid,
        oauth_client: { id, secret }
      } = await this.registerRunner({ name, labels });

      const gpu = await gpuPresent();
      const command = `docker container run -t -a stderr -a stdout --rm \
      -v /var/run/docker.sock:/var/run/docker.sock \
      -v /var/lib/docker/containers:/var/lib/docker/containers:ro \
      -v ${workdir}:/tmp \
```

Reply:
Should be ` ${workdir}` but when I set it the job hangs despite of setting every single permission on earth @iterative/cml any ideas?

Reply:
What were you getting as output?


Off-topic: Do you have feelings about preferring the long-hand version of command line args in code?

Reply:
> What were you getting as output?

NOTHING! Job just hangs forever...

> Off-topic: Do you have feelings about preferring the long-hand version of command line args in code?

I do not understand. Could you please elaborate a bit more?

Reply:
> I do not understand. Could you please elaborate a bit more?

sure using `docker container run`
- `-a` vs `--attach`
- `-t` vs `--tty`
- `-v` vs `--volume`
- `-e` vs `--env`

Reply:
ah! Im mimicking bb instructions
I can change for the shorthands

Reply:
I don't think we have been super consistent about it, but I think it would be good for readability.